docs/dependency-updates.md

```diff
@@ -170,3 +170,12 @@ There are a few scripts that depend on p-queue v6.6.2.  Upgrading this past vers
 
```

Comment:
Any movement on `babel-jest, babel-core`? We've updated ts-jest since that caveat was written